### PR TITLE
Documentation: Tweak grammar and fix some typos

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

This resolves some grammar and typos in the documentation.

Also fixes the extension of some images in after effects docs which used uppercase extension even though files were lowercase extension.

## Testing notes:

1. Proofread the changes

### CODE REVIEW

**Comment by BigRoy** [comment]
I believe this PR will also fix the issues with the documentation failing to build like [here](https://github.com/ynput/OpenPype/actions/runs/4409925937) since it mentions it can't find the uppercased `.PNG` files for after effects docs.

File: website/docs/admin_settings.md [hunk 1]
```diff
@@ -7,12 +7,15 @@ sidebar_label: Working with settings
...
-OpenPype stores all of it's settings and configuration in the mongo database. To make the configuration as easy as possible we provide a robust GUI where you can access and change everything that is configurable
+OpenPype stores all of its settings and configuration in the mongo database. To make the configuration as easy as possible we provide a robust GUI where you can access and change everything that is configurable
 
 **Settings** GUI can be started from the tray menu *Admin -> Studio Settings*.
 
-Please keep in mind that these settings are set-up for the full studio and not per-individual. If you're looking for individual artist settings, you can head to
-[Local Settings](admin_settings_local.md) section in the artist documentation.
+:::important Studio Settings versus Local Settings
+Please keep in mind that these settings are set up for the full studio and not per-individual. If you're looking for individual artist settings, you can head to
+[Local Settings](admin_settings_local.md) section in the documentation.
```
Comment (tokejepsen) [inline website/docs/admin_settings.md:16]: This leads up that looks like an orphan page for the local settings. Maybe have that part of `artist_install` or after that?

**Review by tokejepsen — CHANGES_REQUESTED** [review]

File: website/docs/admin_settings.md (hunk 1 above)
Comment (BigRoy) [reply website/docs/admin_settings.md:16]: So it points to [this](https://openpype.io/docs/admin_settings_local) and yes, it's not pretty. However, I'd actually say that the issue then becomes that the Local Settings are just not correctly documented yet. We should start to do that but can be separate from these small fixes. What do you think?

Comment (tokejepsen) [reply website/docs/admin_settings.md:16]: Cool, if you could move it, I'll eventually get to expanding on the Local Settings from this issue; https://github.com/ynput/OpenPype/issues/4620

Comment (BigRoy) [reply website/docs/admin_settings.md:16]: > Cool, if you could move it

Move it how? :)

Comment (tokejepsen) [reply website/docs/admin_settings.md:16]: Sorry, think the best place might here:

Capture

Comment (tokejepsen) [reply website/docs/admin_settings.md:16]: Yeah, fair. Can move it with the PR.

**Review by tokejepsen — APPROVED** [review]

**Pull request merged**